### DISCUSSION TITLE
Phoniebox als Spotify only Setup

### DISCUSSION THREAD

**Discussion by PanoLolas** [MiczFlor/RPi-Jukebox-RFID#2349] [Q&A]
Hallo,
ich habe für meine Kinder eine Phoniebox gebaut, die aber ausschließlich Spotify Titel/Alben/Playlisten abspielen soll. Leider dauert der Startvorgang - wie in diversen Issues beschrieben - mit der +Spotify Version sehr lange und auch der Startsound kommt lange bevor die Box abspielbereit ist und RFID Karten erkennt und abspielt.

Bei der Analyse des Startvorgangs mittels systemd-analyze blame bzw. critical-chain kontte ich schon diverse Zeitfresser eliminieren, bzw. dafür sorgen, dass der Startsound erst nach dem laden von mopidy abgespielt wird (allerdings immer noch vor dem laden der Spotify-Extension). 
Das führt weiterhin dazu, dass nach ca. 12sec der Startsound kommt, die Box aber noch weitere ca. 15sec. braucht, bis sie einen Spotify-Stream startet.

Ich verfolge daher die Idee, möglichst alle Dienste der Phoniebox nicht zu starten, die für den Spotify-Betrieb nicht zwingend notwendig sind (zB SMP, die WebGUI,...)

Leider habe ich noch nicht so ganz durchdrungen, was dieses Minimalsetup ist, bzw. auf welche DIenste verzichtet werden könnte. Hat jemand schon etwas ähnliches ausprobiert und kann mir hier weiterhelfen?

Klar, dass damit ganz schön viel "Magie" der Software verloren geht. Ggf. könnte man all diese Zusatzdienste ja erst bei Bedarf im Nachgang starten, zB durch das Auflegen einer entsprechenden Karte oder einen GPIO-Button am Gehäuse.

Vielen Dank für eure Hilfe,
PL

PS: ach ja, so sieht die Box bei mir aus :-)
![DSC_4210](https://user-images.githubusercontent.com/58434623/69993374-fa0b5a80-154b-11ea-8d6a-a76b2aa32d24.JPG)
![DSC_4212](https://user-images.githubusercontent.com/58434623/69993394-042d5900-154c-11ea-877f-41928dad3e03.JPG)

**Comment by s-martin** [comment]
Siehe hier: http://splittscheid.de/faqs-zu-meiner-phoniebox/#bootspeedup (German only)
